### DISCUSSION TITLE
No Way To Login On iOS If Community Not Listed?

### DISCUSSION THREAD

**Discussion by kelvin0mql** [mastodon/mastodon#18194] [Q&A] [answered]
Tried to join a ham-radio-related mastadon.radio but could not find (Windows Chrome browser).

Mentioned this to Twitter friend who had told me about it in the first place, and he said "Same - but found it on Google."

So I googled. Found it. Was able to sign up and all good.

OK, so then get the iOS app on iPhone. Try to "simply login" with already known credentials.

Nope, the app **does not allow me to just login**. It presents me with a screen... 
![image](https://user-images.githubusercontent.com/8395186/165954934-601af45d-8d6f-49ed-9360-47be12349601.png)
...which insists I search for and choose a community.

I have already done so, elsewhere. I'm already in. I already have an account.

But my chosen community is somehow mysteriously blocked from this search thing that is not skip-able. Why?

How do I bypass this?

Why has Developer/QA assumed that nobody would ever install this on iOS unless they're **not yet** on the platform?

Early days, sure, that would be 98% correct. But come on - you've never done anything on some **other** device first?

**Comment by ClearlyClaire** [comment, marked as answer]
I think you just made a typo in “mastodon.radio”, typing “mast**a**don” instead of “mast**o**don”.
As far as I know, the iOS app should find the server and let you connect once the correct domain is typed, even if it's not part of the curated list.

**Reply by kelvin0mql** [reply]
Yes, someone within mastodon.radio informed me of same, and got me past that point.

But they also brought up an excellent point - that field that looks for all the world like it's a SEARCH field... **isn't??**

Typing "radio" got me nothing.

Typing "ham" got me nothing.

Typing "amateur" got me nothing.

And the category of "Tech" (or whatever it is)... doesn't list it.

If it's going to have a little magnifying glass in the left edge of the field, which is the universal icon for **search**, it should behave pretty searchy.

If the intended use of that field is "type in exactly the name of the community/server/whatever you already know about" then it should be labeled as such, and should not have a misleading magnifying glass.

**Reply by ClearlyClaire** [reply]
It is a search field, and it searches in the curated list of servers available at https://joinmastodon.org/communities

mastodon.radio is apparently not on it

**Reply by kelvin0mql** [reply]
Aha... excellent. Well that explains it, then. Thank you so very much.

**Reply by kelvin0mql** [reply]
Tried to go back to mastodon.radio to explain to THEM what you just told ME (it was one of them who suggested that the search field ain't really a search field).

They're offline.

It's a "curated list of servers".

Um... yeah... if I was the curator, I wouldn't include 'em either, if they're not doing a good job of chasing nines. ((giggle))
